### PR TITLE
Added message sounds

### PR DESCRIPTION
**Pull request by Olzie-12**
This pull request adds a sounds to recipients of a whisper/reply! Very basic and simple, yet very useful

Sorry! I had to recreate used the wrong branch. my bad. Copy of #219

### CODE REVIEW

**Comment by Olzie-12** [comment]
@jpenilla Oh yeah, you're exactly right. I'll change that now

**Comment by jpenilla** [comment]
A couple of minor things:
- The sound should be nullable, and check for null before sending the sound, so it's an optional config value
- I think `messageSound` is better than `messageSounds` since it's one sound, not a list
- The TODO doesn't need to be included, since it's documented in the issue & PR you opened
- Ideally try not to include unrelated formatting changes, not a big deal here since it's just a couple of newlines, but if it was bigger I'd want to have that fixed before merge

Besides that this looks good to me

**Comment by jpenilla** [comment]
Also, I might be wrong (since GitHub doesn't show this explicitly), but I think you may not have checked this box for your PRs:
image

Checking it allows maintainers to rebase or make changes when merging a PR :)

**Comment by Olzie-12** [comment]
I'll get around to doing that and correct its checked already :)

image

**Comment by jpenilla** [comment]
Ok cool, for some reason GitHub just isn't showing the "update branch" buttons like it does on some other PRs... weird

(not that I need to update it yet)

File: common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java
Comment (jpenilla) [inline common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java]:
```suggestion
    public @Nullable Sound messageSound() {
```

File: common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java
Comment (jpenilla) [inline common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java]:
```suggestion
```

File: common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java
Comment (jpenilla) [inline common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java]:
```suggestion
    private @Nullable Sound messageSound = Sound.sound(
```

**Review by jpenilla — CHANGES_REQUESTED** [review]

**Comment by Olzie-12** [comment]
My bad ahaha, im like half asleep!

**Review by jpenilla — APPROVED** [review]

**Comment by Draycia** [comment]
LGTM!

**Pull request merged**
